Dockerfile

```diff
@@ -23,22 +25,27 @@ RUN apt-get update && apt-get install -y \
     \
     less \
     man \
-    vim
+    vim \
+    \
+    libssl-dev
+
+COPY dist/brainiak-* /mnt/brainiak
+
+WORKDIR /mnt/brainiak
 
-WORKDIR /mnt
+COPY tutorials/tutorials tutorials
 
 RUN set -e \
-    && python3 -m pip install --user -U pip \
-    && python3 -m pip install -U brainiak \
-    && python3 -m pip download --no-deps --no-binary :all: brainiak \
-    && export BRAINIAK_VERSION=$(basename brainiak-* .tar.gz | cut -b 10-) \
-    && tar -xf brainiak-*.tar.gz \
-    && for example in brainiak-$BRAINIAK_VERSION/examples/*/requirements.txt; \
+    && python3 -m pip install --user -U "pip<10" \
+    && python3 -m pip install --user -U . \
+    && python3 -m pip install --user -U -r tutorials/requirements.txt \
+    && for example in examples/*/requirements.txt; \
         do python3 -m pip install --user -U -r $example ; done
 
 RUN echo PATH=\"\$HOME/.local/bin:\$PATH\" >> $HOME/.profile \
     && echo "shell -bash" >> ~/.screenrc
 
-EXPOSE 8888
+EXPOSE 8899
```

Comment:
Didn't we want to stay with port 8888 because that is what Jupiter prints as the URL when it starts? I remember users getting confused by that.

Reply:
This is consistent with the script that we run automatically, so Jupyter will print the right thing:
https://github.com/brainiak/brainiak-tutorials/blob/master/tutorials/run_jupyter_docker.sh#L2